app/Http/Controllers/MainController.php

```diff
@@ -69,7 +69,7 @@ public function showVacancies(Request $request)
             ));
         }
 
-        $vacancies = Vacancy::unblockVacancies()->isActive()->paginate();
+        $vacancies = Vacancy::unblockVacancies()->allVacancies()->isActive()->paginate();
```

Comment:
It's correct?

Reply:
works as correct in local version

Reply:
a lot of options such as block vacancy and so on were added thats why I anticipate handful of bugs regarding new scopes soon

Reply:
There is the same scope added to resume and company